### ISSUE TITLE
Doesn't work on iOS simulator anymore? (iOS 8.3)

### ISSUE THREAD

**Issue by ndbroadbent** [rs/SDAVAssetExportSession#35]
Hello,

I'm trying to use SDAVAssetExportSession in the iOS Simulator, but it's not working any more. I'm calling `encoder!.exportAsynchronouslyWithCompletionHandler()`, but the completion handler is never called. The same code works fine on my device (iPhone 4S running iOS 8).

Anyone else experiencing the same issue?

**Comment by tylerjames** [comment]
Yeah, this is my first time using the library and I can't get it to work on the sim. If I step through `exportAsynchronouslyWithCompletionHandler` the execution seems to just disappear somewhere around the time that it's trying to get the render size. Almost as if it has been deallocated or something. Not a peep about it.
